packages/big-design/src/components/StatefulTable/StatefulTable.tsx

```diff
@@ -86,6 +88,16 @@ const InternalStatefulTable = <T extends TableItem>({
     onSort,
   ]);
 
+  const onDragEnd = useCallback(
+    (from: number, to: number) => {
+      const itemsClone = [...items];
+      const item = itemsClone.splice(from, 1);
+      itemsClone.splice(to, 0, ...item);
```

Comment:
🍹 Could we potentially change this to not rely on mutating an array? Maybe with a combination of `slice` and `spread` operations?